test/library/packages/LAPACK.skipif

```diff
@@ -4,29 +4,27 @@
 Require BLAS and LAPACK modules
 
 Testing BLAS and LAPACK modules requires cray-libsci for compiling. The
-following flags are implicitly passed via the back-end 'cc' wrapper:
+following flags are passed either explicitly by the back-end 'cc' wrapper,
+or explicitly with the LLVM backend:
 
     -I $CRAY_LIBSCI_PREFIX_DIR/include
-    -L $CRAY_LIBSCI_PREFIX_DIR/lib
+    -L $CRAY_LD_LIBRARY_PATH
     -l sci_gnu
 
 Since this is a long-running test and Cray machines guarantee BLAS/LAPACK
-availability, we only test on XC, cray-prgenv-gnu target platform, non-whitebox,
-and non-llvm configurations.
+availability, we only test on XC target platform configurations.
 """
 
 from os import getenv, path
 
 # Make sure we're not on a whitebox
-isRealCray = path.exists('/etc/opt/cray/release/CLEinfo') or path.exists('/etc/opt/cray/release/cle-release')
+isRealCray = (path.exists('/etc/opt/cray/release/CLEinfo') or
+              path.exists('/etc/opt/cray/release/cle-release'))
 
 # Make sure we're on an XC
 isRealXC = isRealCray and getenv('CHPL_TARGET_PLATFORM') == 'cray-xc'
 
-# Only test on cray-prgenv-gnu
-isGNU = 'gnu' in str(getenv('CHPL_TARGET_COMPILER'))
```

Comment:
It might be important to continue to skip this test for PrgEnv-intel and PrgEnv-cray. Of course you could check to see if those work OK.

Reply:
Right, there are those as well. I'm surprised it was only tested on `PrgEnv-gnu` before.